cli/flox/src/main.rs

```diff
@@ -95,10 +94,12 @@ fn main() -> ExitCode {
     let _metrics_guard = Hub::global().try_guard().ok();
 
     // Pass down the verbosity level to all pkgdb calls
-    std::env::set_var(
-        "_FLOX_PKGDB_VERBOSITY",
-        format!("{}", verbosity.to_pkgdb_verbosity_level()),
-    );
+    unsafe {
```

Comment:
(non-blocking, curious) Are these changes in prep for edition 2024?

Reply:
looks like it, yes.
not sure if we need that though, with '24 being still 4 months away at least and these setenv cases likely being automatically fixable with cargo `check --fix`.

Reply:
I'm not actually sure, it was just one of many things the compiler yelled at me for during the refactor so I went ahead and took care of it.